src/components/pages/Profile/EditProfile.js

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import { Form, Input, Button, Radio, Modal, Dropdown } from 'antd';
+
+function EditProfile() {
+  const initialValues = {
+    commitment: '',
+  };
+
+  const [ModalOpen, setModalOpen] = useState(false);
+  const [radio, setRadio] = useState(initialValues);
+
+  const commits = ['1:1 Mentoring', 'Pair Program', 'Neither'];
+
+  //// Event Handlers
+
+  const showModal = () => {
+    setModalOpen(true);
+  };
+
+  const handleCancel = () => {
+    setModalOpen(false);
+  };
+
+  const handleOk = () => {
+    setModalOpen(false);
+  };
+
+  const handleRadio = event => {
+    setRadio(event.target.value);
+  };
+  ////
+
+  return (
+    <>
+      <Button type="primary" onClick={showModal}>
+        Edit
+      </Button>
+      <Modal visible={ModalOpen} onCancel={handleCancel} onOk={handleOk}>
+        <h1>Edit Profile:</h1>
+        <br />
+        <Form
+          name="basic"
+          labelCol={{
+            span: 8,
+          }}
+          wrapperCol={{
+            span: 14,
+          }}
+        >
+          <Form.Item
+            label="Full Name"
+            name="fullName"
+            rules={[
+              {
+                required: true,
+                message: 'Full Name Required!',
+              },
+            ]}
+          >
+            <Input />
+          </Form.Item>
```

Comment:
After meeting today (Tuesday 1/25), it was decided that it would be best to split up instances of "Full Name" inputs into separate inputs for "First Name" and "Last Name" for future use. I'd recommend doing the same thing here to keep things cohesive.